scripts/run-mod-settings.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+set -e
+U=http://localhost:9130
+T=testlib15
+username=testing_admin
+password=admin
+curl -d"{\"id\":\"$T\"}" $U/_/proxy/tenants
+curl -d'{"name":"DB_HOST","value":"localhost"}' $U/_/env
+curl -d'{"name":"DB_PORT","value":"5432"}' $U/_/env
+curl -d'{"name":"DB_USERNAME","value":"postgres"}' $U/_/env
+curl -d'{"name":"DB_PASSWORD","value":"postgres3636"}' $U/_/env
+curl -d'{"name":"DB_DATABASE","value":"postgres"}' $U/_/env
+
+# Set of modules that are necessary to bootstrap admin user
+CORE_MODULES="mod-users mod-login mod-permissions"
+
+TEST_MODULES="mod-settings"
+
+compile_module() {
+	local m=$1
+	if test ! -d $m; then	
+		git clone --recurse-submodules git@github.com:folio-org/$m
+
+	fi
+	if test ! -d $m; then
+		echo "$m missing. git clone failed?"
+		exit 1
+	fi
+	cd $m
+	mvn -DskipTests -Dmaven.test.skip=true verify
+	cd ..
+}
+register_module() {
+	local m=$2
+	echo "Register module $m"
+	local md=$m/target/ModuleDescriptor.json
+	if test ! -f $md; then
+		compile_module $m
+	fi
+	if test ! -f $md; then
+		echo "$md missing pwd=`pwd`"
+		exit 1
+	fi
+	if test "$1" != "x"; then
+		OPT=-HX-Okapi-Token:$1
+	else
+		OPT=""
+	fi
+	curl -s $OPT -d@$md $U/_/proxy/modules -o /dev/null
+	local dd=$m/target/DeploymentDescriptor.json
+}
+
+deploy_module() {
+	local m=$2
+	echo "Deploy module $m"
+	if test "$1" != "x"; then
+		OPT=-HX-Okapi-Token:$1
+	else
+		OPT=""
+	fi
+	local dd=$m/target/DeploymentDescriptor.json
+	curl -s $OPT -d@$dd $U/_/deployment/modules -o /dev/null
+}
+
+deploy_modules() {
+	for m in $2; do
+		register_module $1 $m
+	done
+	for m in $2; do
+		deploy_module $1 $m
+	done
+}
+
+install_modules() {
+	local j="["
+	local sep=""
+	for m in $3; do
+		j="$j $sep {\"action\":\"$2\",\"id\":\"$m\"}"
+		sep=","
+	done
+	j="$j]"
+	if test "$1" != "x"; then
+		OPT=-HX-Okapi-Token:$1
+	else
+		OPT=""
+	fi
+	curl -s $OPT "-d$j" "$U/_/proxy/tenants/$T/install?purge=true&tenantParameters=loadReference%3Dtrue%2CloadSample%3Dtrue"
```

Comment:
If you make this `"-d[$j]"` you can omit the business with the square brackets earlier.